packages/SidePanel/src/hooks/useBodyOverflow.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export default function useBodyOverflow(disableBodyOverflow) {
+  React.useEffect(() => {
+    if (disableBodyOverflow) {
+      document.body.style.overflow = "hidden";
+      return;
+    }
+
+    document.body.style.overflow = "auto";
```

Comment:
I think I got into some trouble with the old SidePanel over this... We had a SidePanel in a Takeover... the Takeover already set the overflow to hidden, but when the SidePanel closed it would restore the overflow to auto and screw up the Takeover.  It's important to restore the overflow as it was, not assume it should be `auto`.

Reply:
https://github.com/acl-services/acl-ui/blob/master/src/components/SidePanel/SidePanel.js#L178